# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1386411462 -3600
#      Sat Dec 07 11:17:42 2013 +0100
# Branch stable
# Node ID a829f40933a8935902adfcb1227719c5f467c5e8
# Parent  523dd79ee726f4873374604a1ac6116555a3f7bb
fix minimized main window after resetting window layout (bug #40832)

* main_window.cc (reset_windows): show normal window after resetting layout

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -655,16 +655,17 @@ main_window::exit (int status)
 }
 
 void
 main_window::reset_windows (void)
 {
   QSettings *settings = resource_manager::get_default_settings ();
 
   set_window_layout (settings);
+  showNormal ();  // make sure main window is not minimized
 }
 
 void
 main_window::change_directory (const QString& dir)
 {
   // Remove existing entry, if any, then add new directory at top and
   // mark it as the current directory.  Finally, update the file list
   // widget.
